src/Notes/CouponPageMoved.php

```diff
@@ -94,15 +94,16 @@ public static function get_note() {
 	 * @return bool
 	 */
 	protected static function has_unactioned_note() {
-		$notes = self::get_data_store()->get_notes(
-			[
-				'name'       => [ self::NOTE_NAME ],
-				'status'     => [ 'unactioned' ],
-				'is_deleted' => false,
-			]
+		global $wpdb;
+
+		$unactioned_note_count = $wpdb->get_var(
```

Comment:
Could we use `Notes::get_note()` to retrieve these notes? 

In general, we should try our best to limit custom queries like this where possible.

Reply:
We could use `Notes::get_note()` if we knew the `$note_id`, but we don't.

Reply:
In that case you can use:

```
		$data_store = Notes::load_data_store();
		$note_ids   = $data_store->get_notes_with_name( self::NOTE_NAME );
		if ( count( $note_ids ) > 0 ) {
				$note = Notes::get_note( $note_ids[0] )
		}
```

Also, not a necessity in this PR, but I would not at all be opposed to having a helper function do this.  We've used this pattern quite a bit of getting note IDs and looping over them; seems like a simply `get_note_by_name()` would not be a bad thing.